packages/react-components/react-theme/src/global/colors.ts

```diff
@@ -848,54 +848,94 @@ const charcoal: ColorVariants = {
   tint60: '#f7f7f7',
 };
 
-export const sharedColors: GlobalSharedColors = {
-  darkRed,
-  burgundy,
-  cranberry,
+export type StatusSharedColors = 'red' | 'green' | 'darkOrange' | 'yellow' | 'berry' | 'lightGreen' | 'marigold';
```

Comment:
I expected the names would be 'available', 'away', 'offline', etc. so that themes could change out the colors to whatever they want.  If we are going with color names, that is OK - just not what I guessed they would be :)

Reply:
IIRC we drew the line at renaming tokens. Any renames will wait for a future major version bump.

Reply:
That was an idea @daisygeng had as well. These are the reasons why we haven't done those changes now:
- We do not want to rename any existing tokens that far in the RC process.
- There are some naming edge cases which need more thinking and discussion (`Red` is used for all **Danger**, **Busy** and Avatar).
- There is no agreement across design.

For that reasons we are not renaming anything now, we plan to re-consider this later and introduce it to v10 (and possibly in a backwards compatible way to v9).